src/main/java/com/strandls/observation/pojo/TopUploadersInfo.java

```diff
@@ -0,0 +1,47 @@
+package com.strandls.observation.pojo;
+
+public class TopUploadersInfo {
+	
+	String name;
```

Comment:
add private access modifiers to all the variables used

Reply:
added private